sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.7.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Early unlocking of encrypted systems via ssh for dracut"
+HOMEPAGE="https://github.com/dracut-crypt-ssh/dracut-crypt-ssh"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-kernel/dracut"
+RDEPEND="${DEPEND}
+	|| ( net-misc/dhcp net-misc/dhcpcd )
```

Comment:
I'm not trying to be annoying here, but genuinely interested:
If these two work, what about other dhcp providers, such as netifrc, connman, networkmanager? I think that should cover them all, well perhaps systemd itself works too?

Reply:
I think those are all valid solutions, systemd should also work via its systemd-networkd